pkg/config/legacy/converter.go

```diff
@@ -15,156 +15,157 @@ import (
 )
 
 // FromAgentConfig reads the old agentConfig configuration, converts and merges
-// the values into the current config.Datadog object
+// the values into the current configConverter object
```

Comment:
```suggestion
// the values into the current configuration object
```
IMO nothing has changed, the global config still gets modified. On top of that `configConverter` has no meaning to someone reading the documentation because it's not part of the function signature.